<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widebtn</class>
 <widget class="QDialog" name="widebtn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="widthline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="widebtn_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ОУКЕЙ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
